bsk_rl/envs/general_satellite_tasking/simulation/dynamics.py

```diff
@@ -856,13 +859,15 @@ def _set_storage_unit(
             priority: Model priority.
             storageUnitValidCheck: If True, check that the storage level is below the
                 storage capacity.
+            setStorageInit: Initial storage level [bits]
         """
         self.storageUnit = simpleStorageUnit.SimpleStorageUnit()
         self.storageUnit.ModelTag = "storageUnit" + self.satellite.id
         self.storageUnit.storageCapacity = dataStorageCapacity  # bits
         self.storageUnit.addDataNodeToModel(self.instrument.nodeDataOutMsg)
         self.storageUnit.addDataNodeToModel(self.transmitter.nodeDataOutMsg)
         self.storageUnitValidCheck = storageUnitValidCheck
+        self.storageUnit.setDataBuffer(setStorageInit)
```

Comment:
Does BSK properly handle range checking (negative storage? setStorageInit>dataStorageCapacity?)? Is there any sort of warning if that's violated? Or do you not think that's necessary

Reply:
The storage module doesn't allow the storage to go lower than zero or higher than the limit using the setDataBuffer. So, in case the data amount passed to setDataBuffer violates one of those cases, the module won't perform the addition/subtraction. Since these cases won't happen, I don't think warnings are really necessary.